samples/interactive-tutorials/src/main/java/product/CrudProduct.java

```diff
@@ -123,8 +124,8 @@ public static Product getProduct(String productName) throws IOException {
     GetProductRequest getProductRequest =
         GetProductRequest.newBuilder().setName(productName).build();
 
-    try {
-      product = ProductServiceClient.create().getProduct(getProductRequest);
+    try (ProductServiceClient serviceClient = ProductServiceClient.create()) {
```

Comment:
Can't comment above but let's probably update the above createProduct method as well.

Reply:
Changed.